pkg/env/env.go

```diff
@@ -64,14 +65,17 @@ func InitConfig() error {
 	viper.AutomaticEnv()
 
 	// If a config file is found, read it in.
-	if err := viperConfig.ReadInConfig(); err != nil {
-		logrus.Debug("Can't read config file, all values will be prompted, err: ", err)
-	}
+	err = viperConfig.ReadInConfig()
 
 	getNestedArgValues()
 
 	promptMissingValues()
 
+	if err != nil {
+		createConfigFile()
```

Comment:
@alexander-demichev I guess we will have more places similar to this one, where we use survey for query user for an input. I would suggest to use some well defined prefix for such queries. So that we know exactly, by the name of the function, that we're asking user. E.g. use "cli prefix" ... cliCreateConfig().